mtags/src/main/scala-3/scala/meta/internal/pc/completions/Completions.scala

```diff
@@ -865,4 +934,114 @@ class Completions(
             else compareByRelevance(o1, o2)
       end compare
 
+  object MatchExtractor:
+    def unapply(path: List[Tree]) =
+      path match
+        // foo mat@@
+        case (sel @ Select(qualifier, name)) :: _
+            if "match".startsWith(name.toString()) && text.charAt(
```

Comment:
It was resolved here https://github.com/scalameta/metals/pull/4229#discussion_r944207169
When we type `.match` the tree changes from `Select` to `Match` and we are not show the match completions for `xxx.ma@@` because I think most users would see it as a bug